cvat-cli/src/cvat_cli/parser.py

```diff
@@ -14,6 +14,8 @@
 
 from .version import VERSION
 
+SSL_VERIFICATION_ENV_VAR = "SSL_VERIFY"
```

Comment:
SSL_NO_VERIFY is more or less standard name for the variable (e.g., conda, git use it with a prefix)

Reply:
I decided to only use `--insecure` arg instead.